json/visitor.go

```diff
@@ -100,6 +101,15 @@ func (v *Visitor) SetIgnoreInvalidFloat(b bool) {
 	v.ignoreInvalidFloat = b
 }
 
+// SetUnambiguousFloat configures how the visitor encodes floating point values to a json float unambiguously.
+// By default, equiv to SetUnambiguousFloat(false), the visitor will encode with the smallest number of digits.
+// e.g. 1.0 to 1 instead of 1.0, 100000000 to 1e+8 instead of 1.0e+8.
+// If true is passed, floating point values will always contain a decimal point,
+// in either decimal form or scientific notation.
+func (v *Visitor) SetUnambiguousFloat(b bool) {
```

Comment:
[To reviewer] feel free to suggest a better name, e.g. SetAlwaysEmitDecimalPoint

Reply:
thanks for the suggestion. Updated.